### PR TITLE
Spc functionality

### PR DESCRIPTION
**Pull request by agudek**

### CODE REVIEW

File: src/modules/super-punch-card.js [hunk 1]
```diff
@@ -1,22 +1,33 @@
...
+            { "start": "2016-06-14T18:18:26.680Z", "end": "2016-06-14T18:19:52.754Z", "session": { "url": "http://146.185.128.124/api/sessions/Travis/thervh70/ContextProject_RDD/149/", "id": 2, "pull_request": { "url": "http://146.185.128.124/api/pull-requests/thervh70/ContextProject_RDD/149/", "repository": { "url": "http://146.185.128.124/api/repositories/thervh70/ContextProject_RDD/", "owner": "thervh70", "name": "ContextProject_RDD", "platform": "GitHub" }, "pull_request_number": 149 }, "user": { "url": "http://146.185.128.124/api/users/Travis/", "id": 1, "username": "Travis" } } },
+        ]
+    ];
+    /* jshint ignore:end*/
+
+    /*jshint ignore: start*/
+    var sessions_data = {
```
Comment (DaanvanderValk) [inline src/modules/super-punch-card.js]: Shouldn't this be removed?

File: src/modules/super-punch-card.js [hunk 2]
```diff
@@ -436,93 +494,236 @@ define(function () {
...
+                    //return color(EVENT_TYPES.indexOf(d.type));
+                    })
+                .attr('stroke-width', 20);
+
+                d3.select('#' + module.name).select('svg')
+                    .transition()
+                    .duration(300)
+                        .attr('viewBox', '0 0 1440 ' + (h + 30 * sessionNumbers.length + 10));
+
+                //var tip2 = d3.tip()
```
Comment (DaanvanderValk) [inline src/modules/super-punch-card.js]: Are you planning to use this?

File: src/modules/super-punch-card.js (hunk 1 above)
Comment (agudek) [inline src/modules/super-punch-card.js]: Will be removed in next PR

File: src/modules/super-punch-card.js (hunk 2 above)
Comment (agudek) [inline src/modules/super-punch-card.js]: Yes, this is going to be used.

**Pull request merged**